app/serializers/alchemy/json_api/element_serializer.rb

```diff
@@ -4,11 +4,12 @@ module JsonApi
     class ElementSerializer
       include FastJsonapi::ObjectSerializer
       attributes(
+        :name,
+        :fixed,
```

Comment:
I don't think we need this one, as the distinction would already be there through the association, don't you think?

Reply:
This is true. But this Serializer might be used by other collections that have the fixed elements included and then it is nice to know, no?

Reply:
OK